src/App.tsx

```diff
@@ -1,25 +1,13 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
```

Comment:
React Typescript docs recommend importing React like so:
import * as React from "react";
although there is nothing inherently wrong with importing normally, I would stick with the futureproof way.
Reference - https://react-typescript-cheatsheet.netlify.app/docs/basic/setup#import-react